src/test/resources/io/managed/services/test/quickstarts/quickstarts.feature

```diff
@@ -25,6 +25,37 @@ Feature: Quick starts for OpenShift Streams for Apache Kafka
     When you create the Kafka topic test-topic
     Then the Kafka topic test-topic is listed in the topics table
 
+  Scenario: Java test
+
+    Given you have a Red Hat account
+    * you are logged in to the OpenShift Streams for Apache Kafka web console
+    * you have a running Kafka instance in OpenShift Streams for Apache Kafka
+
+     # 1. Importing the Python sample code
+    When you clone the rhosak_example_codes repository from GitHub examples for Java
+    Then the rhosak_example_codes example repository for java is available locally
+
+    # 2. Configuring the Quarkus example application to connect to a Kafka instance
+    Given you have the bootstrap server endpoint for your Kafka instance
+    * you have the generated credentials for your service account
+    * you have the OAUTHBEARER token endpoint for the Kafka instance
+    When you set the Kafka instance bootstrap server endpoint, service account credentials, and OAUTHBEARER token endpoint as environment variables for Java
+
+    # 3. Creating the prices Kafka topic in OpenShift Streams for Apache Kafka
+    Given you have a running Kafka instance in OpenShift Streams for Apache Kafka
+    * the Kafka instance is in Ready state
+    When you have created the Kafka topic prices
+    Then the Kafka topic prices is listed in the topics table
+
+    # 4. Running the Java client example producer
+    Given the Kafka instance is in Ready state
+    Given you have set the permissions for your service account to produce and consume from topic prices
+    Then you run Java client example producer
+
+    # 5. Running the Java client example producer
+    Given the Kafka instance is in Ready state
+    Then you run Java client example consumer
```

Comment:
is Running example consumer our end goal ? 
I mean that it is worth either mentioning here that the consumer is also supposed to consume some massages (i can see the implementation), or set is as separated step. 
Any approaches works for me.

Reply:
I have created a separate step.